network-mux/demo/cardano-ping.hs

```diff
@@ -222,6 +222,12 @@ keepAliveReqEnc _ cookie =
 keepAliveReq :: NodeVersion -> Word16 -> ByteString
 keepAliveReq v c = CBOR.toLazyByteString $ keepAliveReqEnc v c
 
+keepAliveDone :: ByteString
+keepAliveDone =
+    CBOR.toLazyByteString $
+         CBOR.encodeListLen 1
+      <> CBOR.encodeWord 2
```

Comment:
You should cover the pre v7 version of the protocol too.

Reply:
Good point

Reply:
Note the shortcoming of the approach of integration of version and version data: we need to pattern match on the version number: when we will add `v8` for `node-to-node` the `cardano-ping` will break if one forgets to update the encoder.

Reply:
I pushed one more commit to fix this.